src/theme/components/menu.ts

```diff
@@ -58,40 +58,41 @@ const Menu = defineMultiStyleConfig({
           },
         },
         actionItem: {
-          outline: "none !important",
-          textColor: "ui.typography.body",
-          paddingY: "xs",
-          paddingX: "s",
           fontSize: "desktop.body.body2",
           fontWeight: "body.body2",
+          lineHeight: "1.5",
+          outline: "none !important",
+          paddingX: "s",
+          paddingY: "xs",
+          textColor: "ui.typography.body",
           _hover: {
-            fontWeight: 510,
             bg: "ui.bg.hover",
+            fontWeight: "medium",
```

Comment:
Not for this PR but "bg" and "fontWeight" are the same for various states. It'd be great to have an object used for multiple states in various components. I imagine this _should_ (?) be a more global style.

Reply:
Yes, there are a lot of repeating styles in there. I created [this ticket](https://newyorkpubliclibrary.atlassian.net/browse/DSD-1999) to look at simplifying the style code.